simp_le.py

```diff
@@ -994,6 +994,10 @@ def create_parser():
         '--server', metavar='URI', default=LE_PRODUCTION_URI,
         help='Directory URI for the CA ACME API endpoint.',
     )
+    http.add_argument(
+        '--ca_bundle', metavar='PATH', default=None,
+        help='Path to alternate CA bundle.',
+    )
```

Comment:
Rather than adding this flag, why not just set the env var in CI? (I feel like I may have asked about this at some point before, memory is fuzzy, sorry if we've covered this).

Reply:
Because I was under the false impression that it did not work and did not even bother to re-check (facepalm again).

Looks like this : https://github.com/buchdag/simp_le/commit/1e956eb26a6a38ca42d670d995b83c5015a7d03e

Given the fact that Pebble recommend that the client have a way to set an alternate trusted CA via CLI, I'm not sure which method we should prefer. Both suits me.